prime-router/metadata/fhir_transforms/default-sender-transform.yml

```diff
@@ -0,0 +1,33 @@
+constants:
+  rsext: '"https://reportstream.cdc.gov/fhir/StructureDefinition/"'
+
+elements:
+  #########################
+  # ReportStream specific #
+  #########################
+# todo:  -//there isn't a way for the name to stay just the CDC prime part and have the namespace id no endpoint set
+  - name: sending-application-namespace-id-no-endpoint
+    condition: true
+    resource: 'Bundle.entry.resource.ofType(MessageHeader)'
+    bundleProperty: '%resource.source.name'
+    value: [ '"CDC PRIME - Atlanta: 2.16.840.1.114222.4.1.237821"' ]
+
+  - name: sending-application-universal-id-with-endpoint
```

Comment:
Do we need the `no-endpoint` `with-endpoint` endings on these names still?

Reply:
Maybe not? I left them to be consistent with the naming in the message-header but can remove them

Reply:
I think given that the sender transform is independent of the fhir-to-hl7 conversion, we should just name these whatever makes the most sense for what these are doing in isolation. So something like `message-header-source-name` and `message-header-source-endpoint` might be my recommendation, but it works out either way, don't feel too strongly about it.

Reply:
That makes sense. I changed them, thank you!